app/src/main/res/values/strings.xml

```diff
@@ -560,12 +560,23 @@ Upload your first media by tapping on the add button.</string>
   <string name="dialog_box_text_nomination">Why should %1$s be deleted?</string>
   <string name="review_is_uploaded_by">%1$s is uploaded by: %2$s</string>
   <string name="default_description_language">Default description language</string>
-  <string name="delete_reason_spam_selfie">A selfie</string>
-  <string name="delete_reason_spam_blurry">Blurry</string>
-  <string name="delete_reason_spam_nonsense">Nonsense</string>
-  <string name="delete_reason_spam_other">Other</string>
-  <string name="delete_reason_copyright_pressphoto">Press photo</string>
-  <string name="delete_reason_copyright_internetphoto">Random photo from internet</string>
-  <string name="delete_reason_copyright_logo">Logo</string>
-  <string name="delete_reason_copyright_other">Other</string>
+  <string name="delete_helper_make_deletion_toast_1">Trying to nominate</string>
+  <string name="delete_helper_make_deletion_toast_2">for deletion </string>
+  <string name="delete_helper_delete_summary_1">Nominating</string>
+  <string name="delete_helper_delete_summary_2">for deletion.</string>
+  <string name="delete_helper_show_deletion_title">Nominating for Deletion</string>
+  <string name="delete_helper_show_deletion_title_if">Success</string>
+  <string name="delete_helper_show_deletion_message_if_1">Successfully nominated</string>
```

Comment:
This string naming is not clear. Additionally, you can insert variables into one string instead of using 2 strings. See the example at line 561:
   <string name="review_is_uploaded_by">%1$s is uploaded by: %2$s</string>